recipe/meta.yaml

```diff
@@ -8,7 +8,7 @@ package:
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 1bc5059e947aec28241d9f7fd6396df8865d99883edb740832f6d281fbe4e2eb
+  sha256: a0ba6518d2035851bc471e74eb9870708a33e4654912149c8bf87702bef9a031
 
 build:
   number: 0
```

Comment:
```suggestion
  number: 0
  skip: true  # [py==36]
```

Reply:
I think we need that; but I can't update the PR I'm no maintainer. 

@asmeurer ?